lib/datasource/index.ts

```diff
@@ -313,15 +313,12 @@ export async function getPkgReleases(
   const versioning =
     config.versioning || getDefaultVersioning(config.datasource);
   const version = allVersioning.get(versioning);
-  // Return a sorted list of valid Versions
-  function sortReleases(release1: Release, release2: Release): number {
-    return version.sortVersions(release1.version, release2.version);
-  }
-  if (res.releases) {
-    res.releases = res.releases
-      .filter((release) => version.isVersion(release.version))
-      .sort(sortReleases);
-  }
+
+  // Filter and sort valid versions
+  res.releases = res.releases
```

Comment:
So you're sure it must be an array?

Reply:
Yes, otherwise it had already failed in next filter statement.